content/docs/guides/flyway-multiple-environments.md

```diff
@@ -0,0 +1,211 @@
+     --url https://console.neon.tech/api/v2/projects/project_id/branches \
+     --header 'accept: application/json' \
+     --header 'authorization: Bearer $NEON_API' \
+     --header 'content-type: application/json' \
+     --data '
+{
+  "branch": {
+    "name": "development"
+  },
+  "endpoints": [
+    {
+      "type": "read_only"
+    }
+  ]
+}
+' | jq
+```
+
+```bash
+neonctl branches create --name development
+```
+
+</CodeTabs>
+
+## Retrieve your Neon database connection strings
+
+From the Neon **Dashboard**, retrieve the connection string for each branch (`main`, `development`, and `staging`) from the **Connection Details** widget. Use the **Branch** drop-down menu to select each branch before copying the connection string.
+
+Your connection strings should look something like the ones shown below. Note that the hostname differs for each (the part starting with `ep-` and ending with `aws.neon.tech`). That's because each branch is hosted on its own compute instance.
+
+- **main**
+
+    <CodeBlock shouldWrap>
+
+    ```bash
+    jdbc:postgresql://ep-cool-darkness-123456.us-east-2.aws.neon.tech/neondb?user=alex&password=AbC123dEf
+    ```
+
+    </CodeBlock>
+
+- **development**
+
+    <CodeBlock shouldWrap>
+
+    ```bash
+    jdbc:postgresql://ep-mute-night-47642501.us-east-2.aws.neon.tech/neondb?user=alex&password=AbC123dEf
+    ```
+
+    </CodeBlock>
+
+- **staging**
+
+    <CodeBlock shouldWrap>
+
+    ```bash
+    jdbc:postgresql://ep-shrill-shape-27763949.us-east-2.aws.neon.tech/neondb?user=alex&password=AbC123dEf
+    ```
+
+    </CodeBlock>
+
+## Configure flyway to connect each environment
+
+To enable Flyway to connect to multiple environments, we'll create a configuration file for each environment and add the environment-specific connection details. When running Flyway, you'll specify the configuration file to be used.
+
+<Admonition type="note">
+By default, Flyway loads its configuration from the default `conf/flyway.conf` file. This is true even if you specify another configuration file when running Flyway. You can take advantage of this behavior by defining non-environment specific configuration settings in the default `conf/flyway.conf` file, and placing your environment-specific settings in separate configuration files, as we'll do here.
+</Admonition>
+
+1. Change to your Flyway `/conf` directory and create the following configuration files, one for each environment, by copying the default configuration file. For example:
+
+    ```bash
+    cd ~/flyway-9.22.3/conf
+    cp flyway.conf env_dev.conf
+    cp flyway.conf env_staging.conf
+    cp flyway.conf env_prod.conf
+    ```
+
+2. In each configuration file, update the following items with the correct connection details for the database environment. The `url` setting will differ for each environment. In this example, where you are the only user, the `user` and `password` settings should be the same for each of your three database environments.
```

Comment:
```suggestion
2. In each configuration file, update the following items with the correct connection details for that database environment. The `url` setting will differ for each environment (in `env_prod.conf` this will point to `main`). In this example, where you are the only user, the `user` and `password` settings should be the same for each of your three database environments.
```